python/eos/data/pypmc.py

```diff
@@ -226,10 +226,8 @@ def create(path, parameters, samples, weights, proposal, compute_test_stat=False
         :type samples: 2D numpy array
         :param weights: Weights on a linear scale as a 2D array of shape (N, 1).
         :type weights: 1D numpy array, optional
-        :param compute_test_stat: (optional) If true, the weighted samples will be used to numerically estimate the inverse CDF of -2 * log(PDF) of the mixture.
-        :type compute_test_stat: bool
-        :param sigma_test: (optional) If compute_test_stat is true, the inverse CDF of -2*log(PDF) will be evaluated for the significance values contained in sigma_test.
-        :type sigma_test: list or iterable
+        :param sigma_test_stat: (optional) If different from [], the inverse CDF of -2*log(PDF) will be evaluated for these significance values.
```

Comment:
``If provided, the inverse CDF of -2*log(PDF) will be evaluated, using the provided values as the respective significance.``